graspologic/nominate/spectralVN.py

```diff
@@ -0,0 +1,327 @@
+    distance based algorithm. In the simple unattributed case, for each seed vertex
+    :math:`u`, the other vertices are ranked in order of euclidean distance from
+    :math:`u`. In the attributed case, vertices are ranked by relatedness to each
+    attribute present in the set of seed vertices.
+
+    Parameters
+    ----------
+    input_graph: bool, default = True
+        Flag whether to expect two full graphs, or the embeddings.
+
+        - True
+            .fit and .fit_predict() expect graphs as adjacency matrix, provided as
+            ndarray of shape (n, n). They will be embedded using the specified embedder.
+        - False
+            .fit() and .fit_predict() expect an embedding of the graph, i.e. a ndarray
+            of size (n, d).
+    embedder: str or BaseEmbed, default = 'ASE'
+        May provide either a embed object or a string indicating which embedding method
+        to use, which may be either:
+        "ASE" for :py:class:`~graspologic.embed.AdjacencySpectralEmbed` or
+        "LSE" for :py:class:`~graspologic.embed.LaplacianSpectralEmbed`.
+
+    Attributes
+    ----------
+    attribute_labels_ : np.ndarray
+        The attributes of the vertices in the seed (parameter 'y' for fit). Shape is the
+        number of seed vertices. Each value is unique in the unattributed case.
+    unique_attributes_ : np.ndarray
+        Each unique attribute represented in the seed. One dimensional. In the
+        unattributed case of SVN, the number of unique attributes (and therefore the
+        shape along axis 0 of `unique_att_`) is equal to the number of seeds ( the shape
+        along axis 0 of `attribute_labels_`).
+    distance_matrix_ : np.ndarray
+        The euclidean distance from each seed vertex to each vertex. Shape is
+        ``(number_vertices, k)`` if attributed or shape is
+        ``(number_vertices, number_seed_vertices)`` if unattributed.
+
+    References
+    ----------
+    .. [1] Fishkind, D. E.; Lyzinski, V.; Pao, H.; Chen, L.; Priebe, C. E. Vertex
+        nomination schemes for membership prediction. Ann. Appl. Stat. 9 2015.
+        https://projecteuclid.org/euclid.aoas/1446488749
+
+    .. [2] Jordan Yoder, Li Chen, Henry Pao, Eric Bridgeford, Keith Levin,
+        Donniell E. Fishkind, Carey Priebe, Vince Lyzinski, Vertex nomination: The
+        canonical sampling and the extended spectral nomination schemes, Computational
+        Statistics & Data Analysis, Volume 145, 2020.
+        http://www.sciencedirect.com/science/article/pii/S0167947320300074
+
+    """
+
+    def __init__(
+        self, input_graph: bool = True, embedder: Union[str, BaseSpectralEmbed] = "ASE"
+    ):
+        super().__init__(multigraph=False)
+        self.embedder = embedder
+        self.input_graph = input_graph
+        self.neighbor_inds_ = None
+        self.attribute_labels_ = None
+        self.unique_attributes_ = None
+        self.distance_matrix_ = None
+        self.embedding_ = None
```

Comment:
remove (see for example https://github.com/scikit-learn/scikit-learn/blob/0fb307bf39bbdacd6ed713c00724f8f871d60370/sklearn/decomposition/_pca.py#L323) I think this matters for the purposes of code that checks `hasattr(object, "neighbor_inds_")` as an "is fitted" check